netlify.toml

```diff
@@ -21,12 +21,33 @@
 # Update to the latest version number
 # Enable permanent links to a release (link persists after release is archived)
 [[redirects]]
-  from = "/v0.13-docs/*"
+  from = "/v0.14-docs/*"
   to = "/docs/:splat"
   status = 200
 
 # Update to include the recently archived version
 # Redirects all previously published versions to the latest
+
+[[redirects]]
+  from = "/v0.13-docs/*"
+  to = "/docs/"
+  status = 301
+
+[[redirects]]
+  from = "/v0.12-docs/*"
+  to = "/docs/"
+  status = 301
+  
+[[redirects]]
+  from = "/v0.11-docs/*"
+  to = "/docs/"
+  status = 301
+  
+ [[redirects]]
```

Comment:
We only want to add this .10 redirect for this release (because it wont/shouldnt be published on knative.dev - only available through the GitHub repo from here on).